Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml

```diff
@@ -37,6 +37,17 @@ query: |
     | where ( CommandLine has_any("/q /c color f7&", "Net.We$()bClient", "$b,15,$b.Length-15")) or (CommandLine has "FromBase64String" and CommandLine has_all("-nop", "iex", "(iex"))
     | project TimeGenerated, Computer, NewProcessName, ParentProcessName, Account, NewProcessId, Type, CommandLine
     | extend timestamp = TimeGenerated, HostCustomEntity = Computer , AccountCustomEntity = Account, ProcessCustomEntity = NewProcessName
+    ),
+    (WindowsEvent
+    | where EventID == '4688' and (EventData has_any("/q /c color f7&", "Net.We$()bClient", "$b,15,$b.Length-15") or (EventData has "FromBase64String" and EventData has_all("-nop", "iex", "(iex")) )
+    | extend CommandLine = tostring(EventData.CommandLine) 
+    | where ( CommandLine has_any("/q /c color f7&", "Net.We$()bClient", "$b,15,$b.Length-15")) or (CommandLine has "FromBase64String" and CommandLine has_all("-nop", "iex", "(iex"))
+    | extend NewProcessName = tostring(EventData.NewProcessName)
+    | extend ParentProcessName = tostring(EventData.ParentProcessName)
+    | extend Account =  strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
```

Comment:
Shouldn't we check for an empty domain (and probably also "-") and if so not add "\\"?

Reply:
@oshezaf Done. Should I add it everywhere else?

Reply:
I think so, but maybe phase 2, as it is a lot of work.